modules/macstadium_infrastructure/dhcp-server.tf

```diff
@@ -0,0 +1,50 @@
+resource "vsphere_virtual_machine" "dhcp_server" {
+  name       = "dhcp-server-${var.index}"
+  folder     = "${vsphere_folder.internal_vms.path}"
+  vcpu       = 2
+  memory     = 4096
+  datacenter = "${var.datacenter}"
+  cluster    = "${var.cluster}"
+  domain     = "macstadium-us-se-1.travisci.net"
+
+  network_interface {
+    label = "${var.internal_network_label}"
+  }
+
+  network_interface {
+    label              = "${var.jobs_network_label}"
+    ipv4_address       = "${cidrhost(var.jobs_network_subnet, 10 + var.index)}"
+    ipv4_prefix_length = "18"
+  }
+
+  disk {
+    template  = "${vsphere_folder.vanilla_vms.path}/${var.vanilla_image}"
+    datastore = "${var.datastore}"
+  }
+
+  connection {
```

Comment:
Is this `connection` used?

Reply:
yep! this connection is used by the `macstadium_dhcp` module's `remote-exec`.

Reply:
How? :grin:  I thought `macstadium_dhcp` was having ssh user and host injected as variables.